pkg/networks/usernet/dns.go

```diff
@@ -0,0 +1,84 @@
+// From https://raw.githubusercontent.com/abiosoft/colima/v0.5.5/daemon/process/gvproxy/dnshosts.go
```

Comment:
License notice has to b included https://github.com/abiosoft/colima/blob/v0.5.5/LICENSE

I also expect this to be a separate go pkg